packages/altair/meta.yaml

```diff
@@ -19,7 +19,5 @@ requirements:
     - typing-extensions
     - Jinja2
     - jsonschema
-    - numpy
-    - pandas
-    - toolz
     - packaging
+    - narwhals
```

Comment:
I think this PR would need to wait until #5121 is merged.

Reply:
#5121 is merged. Can we re-run the failing CI?🙏

Reply:
I retriggered by merging in the main branch after fixing a conflict, but it seems like the macos14 selenium test is still failing for what seems to be an unrelated reason:

![image](https://github.com/user-attachments/assets/a586f663-05e4-4259-a4b5-5af0b75df2e6)

Reply:
It is an upstream selenium issue, so please ignore it.